app/models/concerns/logging.rb

```diff
@@ -108,13 +108,23 @@ def self.generate_action(subject_class, attr, action_key, old_value, new_value)
     text
   end
 
+#  def self.action_text(key)
+#    case key
+#    when :update then 'geändert'
+#    when :create then 'angelegt'
+#    when :added then 'hinzugefügt'
+#    when :removed then 'entfernt'
+#    else raise "Unknown action_key #{key}"
+#    end
+#  end
+
   def self.action_text(key)
     case key
-    when :update then 'geändert'
-    when :create then 'angelegt'
-    when :added then 'hinzugefügt'
-    when :removed then 'entfernt'
-    else raise "Unknown action_key #{key}"
+    when :update then  '<%= I18n.t("logging.action_text_update")  %>'
+    when :create then  '<%= I18n.t("logging.action_text_create")  %>'
+    when :added then   '<%= I18n.t("logging.action_text_added")   %>'
+    when :removed then '<%= I18n.t("logging.action_text_removed") %>'
+    else raise "<%= I18n.t('logging.action_text_unknown') %> action_key #{key}"
```

Comment:
```suggestion
    else raise "Unknown action_key #{key}"
```
Das ist eine Exception, die nicht angepasst / übersetzt werden soll, daher hier bitte beim Ursprung bleiben.

Reply:
Erledigt.